### PR TITLE
Tools no docker

### PR DESCRIPTION
**Pull request by pjaol**
Replacing tools utils, removing docker

This provides a quick cli to assist with encryption needs for private git hub accounts
Includes documentation.

### CODE REVIEW

File: tools/src/main/java/com/capitalone/dashboard/tools/App.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+{
+	
+	Options options = new Options();
+    public static void main( String[] args )
+    {
+        App app = new App();
+        CommandLine line = null;
+        try {
+			line = app.parseArgs(args);
+		} catch (ParseException e) {
```
Comment (jdamick) [inline tools/src/main/java/com/capitalone/dashboard/tools/App.java]: looks like tabs / spaces? - alignment issue

File: tools/src/main/java/com/capitalone/dashboard/tools/App.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import com.capitalone.dashboard.tools.utils.EncryptionTool;
+import com.capitalone.dashboard.util.EncryptionException;
+
+/**
+ * Hello world!
```
Comment (jdamick) [inline tools/src/main/java/com/capitalone/dashboard/tools/App.java]: please remove

File: tools/src/main/java/com/capitalone/dashboard/tools/App.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        App app = new App();
+        CommandLine line = null;
+        try {
+			line = app.parseArgs(args);
+		} catch (ParseException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+        if (! line.iterator().hasNext()){
```
Comment (jdamick) [inline tools/src/main/java/com/capitalone/dashboard/tools/App.java]: why is this outside the try/catch?

File: tools/src/main/java/com/capitalone/dashboard/tools/App.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	Options options = new Options();
+    public static void main( String[] args )
+    {
+        App app = new App();
+        CommandLine line = null;
+        try {
+			line = app.parseArgs(args);
+		} catch (ParseException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```
Comment (jdamick) [inline tools/src/main/java/com/capitalone/dashboard/tools/App.java]: probably should fail / exit right? not much can be done in this case. a more meaningful message would be better too..

File: tools/src/test/java/com/capitalone/dashboard/tools/AppTest.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
+package com.capitalone.dashboard.tools;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (jdamick) [inline tools/src/test/java/com/capitalone/dashboard/tools/AppTest.java]: can you add some tests please

**Comment by pjaol** [comment]
Tidied up the changes, was sloppy on my behalf - please review

**Pull request closed without merging**